src/Aspire.Hosting/Dcp/DcpOptions.cs

```diff
@@ -12,6 +12,7 @@ internal sealed class DcpOptions
     private const string DcpCliPathMetadataKey = "DcpCliPath";
     private const string DcpExtensionsPathMetadataKey = "DcpExtensionsPath";
     private const string DcpBinPathMetadataKey = "DcpBinPath";
+    private const string DashboardPathMetadataKey = "aspiredashboardpath";
```

Comment:
Different casing from other metadata keys triggers my OCD. Is there a reason?

Reply:
Provide this feedback on this PR :)

https://github.com/dotnet/aspire/pull/1705

I'm just lining up with what is over on that PR.